src/run.ts

```diff
@@ -162,7 +166,9 @@ export async function runPublish({
 
 const requireChangesetsCliPkgJson = (cwd: string) => {
   try {
-    return require(resolveFrom(cwd, "@changesets/cli/package.json"));
+    return require(require.resolve("@changesets/cli/package.json", {
```

Comment:
I'm not sure why we were using `resolveFrom`. `require.resolve` supported this for a long time already. It also just seems like using `require` here is an easier option than dancing around this using `import-meta-resolve`.